### ISSUE TITLE
Problematic second colon in shortcode for ☀️ (:sunny:️)

### ISSUE THREAD

**Issue by zarko-tg** [joypixels/emojione#644]
We've found by accident that (at least one type of) emoji yields a shortcode with a non-standard closing colon.

emojione.sprites = true
emojione.ascii = true

emojione.toShort('☀️') will return 
![screenshot 2019-02-20 at 09 50 18](https://user-images.githubusercontent.com/6142411/53078649-f3e44a00-34f4-11e9-8ff6-2494d7523063.png)
(Pay close attention to visual difference between the first and the second colon)

Such an inconsistent format / non-ASCII colon is causing certain troubles in our remote systems so we're forced to remove hex efb88f from the shortcode so that only a regular second colon remains in the string.

This does not seem to be the case for other emoji shortcodes.
Is it so on purpose or is this a bug?

We're on emojione 4.5.0

**Comment by zensasian85** [comment]
This is confirmed.

We also experienced same issue, but it's not only the sun. The emoji "heart" has the same issue well.
We are wondering what over emojis have the same issue.
Though, we only experienced this issue on iOS. Android has no problem.

For one app, we are using version 4.0.0, and for an older app, we are using version 3.1.7

Please look into this.

**Comment by caseyahenson** [comment]
Great find @zarko-tg, we'll get a fix for this put out soon

**Comment by dkasak** [comment]
Incidentally, the definitions for those emojis appear to have the wrong fully-qualified sequence specified: they are missing the [VARIATION SELECTOR+16](https://www.fileformat.info/info/unicode/char/fe0f/index.htm), i.e. 0xFE0F == 65039, to make them a proper fully-qualified sequence.

```
   "name": "red heart",
...
      "fully_qualified": "2764",
...
      "default_matches": [
        "2764",
        "2764-fe0f"
      ],
```

```
    "name": "sun",
...
      "fully_qualified": "2600",
...
      "default_matches": [
        "2600",
        "2600-fe0f"
      ],
```

This seems probably related.

**Comment by stephenreddek** [comment]
@caseyahenson Do you have any update on timeline for a patch release?

**Comment by caseyahenson** [comment]
@stephenreddek looking like by July 10 for this fix (along with the version 5.0 update), and will likely require a migration to the https://www.github.com/joypixels/emoji-toolkit library as we'll be deprecating this repo starting with 5.0.

**Comment by stephenreddek** [comment]
@caseyahenson Great. Thanks so much for the update!

**Comment by caseyahenson** [comment]
This is fixed in the 5.0 release https://github.com/joypixels/emoji-toolkit/releases/tag/5.0.0.
